ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/privacy/PrivateTransactionReceiptResult.java

```diff
@@ -30,6 +32,13 @@
   "from",
   "to",
   "output",
+  "publicHash",
+  "privateHash",
```

Comment:
In the JI, we are calling this `transactionHash`. Did we decide to change or was this a mistake?

Reply:
We were calling this publicHash and privateHash internally which is how this came about. Will fix.